nb2/service/person_service.py

```diff
@@ -29,6 +29,23 @@ def get_person_by_slack_user_id(slack_user_id: str):
     return Person.query.filter_by(slack_user_id=slack_user_id).one_or_none()
 
 
+def get_person_name_by_slack_user_id(slack_user_id: str):
+    """
+    Return Person's name with slack_user_id if it exists.
+
+    Required Args:
+        slack_user_id: string representing the Person's primary Slack id.
```

Comment:
[Nit] Would probably be nice to have the same definition for the same args across the app.

I think the other one you used sounds more concise: `The unique Slack identifier for a Person.`

Or maybe a combination of both: "string representing the unique Slack identifier for a Person"

Reply:
I wonder if there is a tool to enforce this/make it easier. Something that parses docstrings and automatically fills in predefined descriptions of parameters.

Reply:
That'd be very interesting. I might look into that